### DISCUSSION TITLE
Strong named dlls, problem with registering

### DISCUSSION THREAD

**Discussion by PontusLindberg** [PrismLibrary/Prism#3050] [WPF] [answered]
Hi!

We are working with add-ins to a CAD application. We have two different add-ins in two different folders. These two add-ins are loading a common dll that exists in each add-in direcory. We use this method in Prism:

containerRegistry.RegisterDialog<xxxView, xxxViewModel>();

The common dll is strong named, and the versions are different.

The issue for us seems to be that Prism will load only the first dll. How can we register our second view/viewmodel successfully?

Kind regards,  Pontus

**Comment by brianlagunas** [comment, marked as answer]
It's not Prism that is loading the assemblies in your CAD extension, it's the CAD application itself. You'll have this problem regardless of if you use Prism or not. You should reach out to the CAD application devs and ask how to handle this scenario.
